doc/whats_new.rst

```diff
@@ -64,6 +64,7 @@ BUG
 
 API
 ~~~
+    - Adds new function :func:`mne.time_frequency.csd_array` to compute the cross-spectral density of multivariate signals stored in an array. Also renamed :func:`mne.time_frequency.csd_compute_epochs` to :func:`mne.time_frequency.csd_epochs` for consistency with other names. :func:`mne.time_frequency.csd_epochs` is now deprecated and will be removed in mne 0.14, by `Nick Foti`_
```

Comment:
It just occurred to me that you also need to add a URL at the bottom of the RST for your name to render properly as a link, see other examples